apps/web/src/pages/settings/WebhookPage/WebhookClaimStatusDisplay.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  IconOutlineDomainVerification,
+  IconOutlineHourglassTop,
+  IconOutlineWebAssetOff,
+  IconRefresh,
+  When,
+} from '@novu/design-system';
+import { LocalizedMessage } from '@novu/shared-web';
+import { FC } from 'react';
+import { IconButton } from '../../../components';
+import { cva } from '../../../styled-system/css';
+import { HStack } from '../../../styled-system/jsx';
+import { Text } from './WebhookPage.shared';
+import { type WebhookClaimStatus } from './WebhookPage.types';
+
+const WEBHOOK_STATUS_LABEL_LOOKUP: Record<WebhookClaimStatus, LocalizedMessage> = {
+  unclaimed: 'Not claimed',
+  pending: 'Pending...',
+  claimed: 'Claimed',
+};
+
+const WEBHOOK_STATUS_ICON_LOOKUP: Record<WebhookClaimStatus, React.ReactNode> = {
+  unclaimed: <IconOutlineWebAssetOff />,
+  pending: <IconOutlineHourglassTop />,
+  claimed: <IconOutlineDomainVerification />,
+};
+
+interface IWebhookClaimStatusProps {
+  status: WebhookClaimStatus;
+  handleRefresh: () => Promise<any>;
+  isLoading: boolean;
+}
+
+const iconTextRecipe = cva({
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Here is my first time using `cva` (an atomic recipe)! It is a great way to have "conditional" styles while still allowing Panda to statically analyze our code and generate the necessary styles.‏